assets/prototype/domain/eventEditor/containers/mutations/useDeleteTicketMutation.js

```diff
@@ -1,11 +1,13 @@
-import get from 'lodash/get';
+import { path } from 'ramda';
```

Comment:
if you only need the one function you can import directly too:

```suggestion
import path from 'ramda/src/path';
```